commonmark-ext-styles/src/main/java/org/commonmark/ext/styles/internal/StylesDelimiterProcessor.java

```diff
@@ -30,22 +37,95 @@ public int getDelimiterUse(DelimiterRun opener, DelimiterRun closer) {
 
     @Override
     public void process(Text opener, Text closer, int delimiterCount) {
-        StringBuilder styleString = new StringBuilder();
+        // Check if the attributes can be applied - if the previous node is an Image, and if all the attributes are in
+        // the set of SUPPORTED_STYLES
+        if (opener.getPrevious() instanceof Image) {
+            boolean canApply = true;
+            List<Node> toUnlink = new ArrayList<>();
+
+            StringBuilder styleString = new StringBuilder();
+            Node tmp = opener.getNext();
+            while (tmp != null && tmp != closer) {
+                Node next = tmp.getNext();
+                // Only Text nodes can be used for attributes
+                if (tmp instanceof Text) {
+                    String styles = ((Text) tmp).getLiteral();
+                    for (String s : styles.split("\\s+")) {
```

Comment:
Split on `\s+` to support the case where more than one space is used to separate the attributes